src/template/volumes/volume1/notebook-test2/nb2.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
can we change the directory name, not to be like some test? It is currently 'notebook-test2' but maybe better to be changed to different name

Reply:
Good catch! This folder shouldn't be here since we're using you zenodo api to download notebooks